numpy/core/memmap.pyi

```diff
@@ -0,0 +1,5 @@
+from typing import List
+
+from numpy import memmap as memmap
+
+__all__: List[str]
```

Comment:
It's not strictly necessary to add stub file here, but it does make it easier to find the `memmap` annotations if someone is looking for them.